src/seqtk/seqtk_sample/config.vsh.yaml

```diff
@@ -0,0 +1,54 @@
+name: seqtk_sample
+namespace: seqtk
+description: Subsamples sequences from FASTA/Q files.
+keywords: [tag1, tag2]
+links:
+  repository: https://github.com/lh3/seqtk/tree/v1.4
+license: MIT
+
+argument_groups:
+  - name: Inputs
+    arguments:
+      - name: --input
+        type: file
+        description: The input FASTA/Q file.
+        required: true
+
+  - name: Outputs
+    arguments:
+      - name: --output
+        type: file
+        description: The output FASTA/Q file.
+        required: true
+        direction: output
+
+  - name: Options
+    arguments:
+      - name: --seed
+        type: integer
+        description: Seed for random generator.
+        default: 42
+      - name: --fraction_number
+        type: double
+        description: Fraction or number of sequences to sample.
+        default: 0.1
+      - name: --two_pass_mode
+        type: boolean
```

Comment:
maybe `boolean_false`? if memory is not an issue, it would just make it slower

Reply:
?

with `boolean_true`, if somebody runs `./seqtk_sample --two_pass_mode`, then `par_two_pass_mode` will be equal to "true", otherwise "false". Is that not what you want?